### PR TITLE
Remove unused enchants plus allow loading of relics after enchantments are loaded. 

### LINKED ISSUE #61701: Artifact passive  "MAX_HP" is not working
### Describe the bug

The extra hp show, but the hit point reset back to is original number after a turn. 

### Steps to reproduce

Example: you have 100 hp plus the bonus an artifact with this passive give like 110 in total, heal from any source, you got 105 hp. A few moment later you got 100 again.
You need : an artifact with MAX_HP and some way to fast heal.

### Expected behavior

The normal extra HP bonus without losing it few moment later.

### Screenshots

_No response_

### Versions and configuration

Windows 7
cdda-windows-tiles-x64-msvc-2022-10-11-1637

### Additional context

Still single.

### LINKED ISSUE #63195: Enchantments increasing Max HP continually reset to pre-enchanted value
### Describe the bug

1. When applying an enchant that increases HP through the MAX_HP ID value, HP resets to the pre-enchant base every few of in-game minutes. 

### Steps to reproduce

1) Start a world with Magiclysm
2) Debug in the Gaia's Chosen attunement, which uses an enchantment attached to it to increase max HP by 30%.

>   "values": [ { "value": "MAX_HP", "multiply": 0.3 } ], )

3) Debug in the Restoration druid spell and cast it
4) Watch as HP heals up slightly toward the new value but then resets down to pre-enchantment max

### Expected behavior

An enchantment that says it increases Max HP should allow HP to actually reach the new Max HP.

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19044.2486 (21H2)
- Game Version: 950acde [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Magiclysm [magiclysm]
]

### Additional context

_No response_

### LINKED ISSUE #63596: Enchantment PAIN has no impact
### Describe the bug

As far as I can tell the enchantment PAIN has no impact.  I've set enchantments using both negative and positive numbers and the enchanted character continues to heal from 44 pain to 32 pain in 1 hour no matter how I adjust the enchantment and reset the world.

### Attach save file

[Aftershock Injection test-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/10736685/Aftershock.Injection.test-trimmed.tar.gz)

### Steps to reproduce

Create a world with aftershock
debug in a cbm testing character
install chemical injection rig cbm 
cause 44 pain 
turn on pain suppression cbm power
wait 1 hour
see 32 pain remains.
Test in different ways with different values in `{
    "id": "afs_healing_injection",
    "type": "bionic",
    "name": { "str": "Pain Suppression Injection System" },
    "description": "The chemical injection system and microcomputer that when activated gives a burst of chemicals that allows your blood to work a little harder at stabilizing the body and then flushes your system with a powerful anesthetic that allows you to continue operating at high capacity.  Upon deactivation, the user may experience extreme lethargy, sudden loss of consciousness, among other symptoms that may prove potentially fatal if not in a safe location.",
    "flags": [ "BIONIC_TOGGLED" ],
    "enchantments": [
      {
        "condition": "ACTIVE",
        "values": [
          { "value": "ARMOR_BIO", "add": 7 },
          { "value": "ARMOR_COLD", "add": 5 },
          { "value": "PAIN", "add": 450 },
          { "value": "METABOLISM", "multiply": 1.5 },
          { "value": "REGEN_HP", "multiply": 2 }
        ]
      }
    ],
    "deactivated_eocs": [ "afs_eoc_hangover_healing" ],
    "act_cost": "100 J",
    "react_cost": "20 J",
    "time": "1 m"
  },`
See that nothing you adjust changes anything regarding pain rate.

### Expected behavior

I would expect that something would impact the pain rate either negatively or positively.

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22621.1105 (22H2)
- Game Version: 88fbd38 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Aftershock [aftershock]
]

### Additional context

Could be solved by addressing #62875

### LINKED ISSUE #63616: Invalid enum string 'SIGHT_RANGE'
### Describe the bug

 DEBUG    : Failed to read from "./save/Magic Island/#TmlraQ==.sav": Invalid enum string 'SIGHT_RANGE' for 'enum enchant_vals::mod'

 FUNCTION : read_from_file
 FILE     : D:\a\Cataclysm-DDA\Cataclysm-DDA\src\cata_utility.cpp
 LINE     : 422
 VERSION  : f0927ac

Caused by #63562

### Attach save file

[Magic Island-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/10755061/Magic.Island-trimmed.tar.gz)

### Steps to reproduce

1. Make a save with base game and Magicalysm mod on game version prior to #63562
2. Spawn and equip any enchanted item.
3. Update the game
4. Load saved game

### Expected behavior

Game loads

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.2604 (22H2)
- Game Version: 170527d [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Blaze Industries [blazeindustries],
    Alternative Map Key [alt_map_key],
    Stats Through Kills [stats_through_kills],
    Magiclysm [magiclysm],
    Sky Island [skyisland]
]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Ramza13**
#### Summary
None

#### Purpose of change
Fix #63596
Fix #61701
Fix #63195
Fix several enchantments.
Clear unused code

#### Describe the solution
Fix several enchantments. 
Removed the remaining non implemented enchantments.

### CODE REVIEW

**Comment by Maleclypse** [comment]
PAIN would be very nice to have if it's an easy fix.  Since otherwise I'll have to figure out how to rework that bionic that used it as it's main function.

**Comment by GuardianDll** [comment]
Btw, can HP_MAX and HP regen could be fixed? They doesn't work aslo, but used in few items here and there

File: data/json/bionics.json [hunk 1]
```diff
@@ -1126,7 +1126,7 @@
     "id": "bio_memory",
     "type": "bionic",
     "name": { "str": "Enhanced Memory Banks" },
-    "description": "Your memory has been enhanced with small quantum storage drives.  While active, you learn - both through reading and practice - more quickly.  Additionally, you forget skills you've learned slower.",
+    "description": "Your memory has been enhanced with small quantum storage drives.  While active, you learn - both through reading and practice - more quickly.",
```
Comment (dseguin) [inline data/json/bionics.json:1129]: Could you separate this string change to get this in during string freeze? Or even better would be to add the try-catch block as a separate PR

**Comment by waveyl** [comment]
This debug message won't point out what enchantment has been removed, this is inconvenient for fixing bugs.

**Comment by Ramza13** [comment]
> This debug message won't point out what enchantment has been removed, this is inconvenient for fixing bugs.

Good point, I just updated #63678 which is where the latest version of that fix is based on your feedback.  Thanks

**Comment by Ramza13** [comment]
> Btw, can HP_MAX and HP regen could be fixed? They doesn't work aslo, but used in few items here and there

I just tested HP_MAX via the GAIA'S CHOSEN attunement and it looked like it worked.  Can you give me some more details on whats wrong with it?

**Comment by GuardianDll** [comment]
So i just checked, and found this:
#61701 seems to be fixed now, but #63195 is - if you have enchantment, that increase your hp, you can't regen the HP beyond the default value (if you have 84 hp, and pick gaia's chosen, you can't regen more than 85)

**Comment by Dan1ss1mo** [comment]
I think jsonifing effect of NOPAIN trait would also be nice, so there would be non hardcoded way to add total pain immunity

**Comment by GuardianDll** [comment]
How often the base pain fade is ticking? Seems like important information to add into `PAIN_REMOVE` doc

**Comment by alp7292** [comment]
what is currently work in progress

**Pull request merged**